signac/__main__.py

```diff
@@ -1469,8 +1469,10 @@ def main():
     parser_find = subparsers.add_parser(
         "find",
         description="""All filter arguments may be provided either directly in JSON
```

Comment:
I think it could be helpful to use some intentional line breaks in this message. I looked at `man` pages for a few executables and commands like `apt search --help`, `docker run --help`. I'd recommend the following:

```
Filter arguments may be provided directly as JSON or in a simplified form using the query API. For, example the following commands are equivalent:

$ signac find key 42
$ signac find '{"a": 42}'

For more information, see: https://docs.signac.io/en/latest/query.html#simplified-filter
```
Note that we have an anchor in the source that can be used to shorten this URL: https://github.com/glotzerlab/signac-docs/blame/ddd4a06b23f82b722303d31b3d1eb7338a2fcd97/docs/source/query.rst#L251

I'd do something similar for the other case, too. This is helpful, thanks!

Reply:
Good idea. The included `argparse.RawDescriptionHelpFormatter` and `argparse.RawTextHelpFormatter` don't allow wrapping of some text while preserving blank lines. There is a [long running feature request for a better formatter for argparse](https://github.com/python/cpython/issues/57015).

I propose adding a new dependency [argparse-formatter](https://pypi.org/project/argparse-formatter/) and its `FlexiFormatter`, which has been [proposed for inclusion in Python](https://github.com/python/cpython/pull/22129).

See the [source of the formatter](https://github.com/davesteele/argparse_formatter) for docs.